apps/shared/avifpng.c

```diff
@@ -330,6 +336,11 @@ avifBool avifPNGRead(const char * inputFilename,
         } else {
             avif->depth = 12;
         }
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+        if (useYCgCoR) {
+            avif->depth += (avif->matrixCoefficients == AVIF_MATRIX_COEFFICIENTS_YCGCO_RE) ? 2 : 1;
+        }
+#endif
```

Comment:
Done

Reply:
I think lines 339-355 should be moved after line 356 so that they are outside the `if (avif->depth == 0) { ... }` block.

I.e., the code should look like the following:

```
    avif->depth = requestedDepth;
    if (avif->depth == 0) {
        if (imgBitDepth == 8) {
            avif->depth = 8;
        } else {
            avif->depth = 12;
        }
    }
#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
    if (useYCgCoR) {
        ...
    }
#endif
```